SABnzbd.py

```diff
@@ -1571,6 +1571,9 @@ def main():
             if not sabnzbd.check_all_tasks():
                 autorestarted = True
                 sabnzbd.TRIGGER_RESTART = True
+            # Sort by completed status
+            if sabnzbd.cfg.auto_sort() and sabnzbd.cfg.auto_sort().split()[0] == "completed":
```

Comment:
Why does this one get special treatment of being run often?
Shouldn't we then just always run this for _all_ auto-sorters?

Reply:
Let's move this to `NzbQueue` and call it from:
https://github.com/puzzledsab/sabnzbd/blob/00e1fb61aa8de9f9e0bbea7dd1de61d3bdc2bd1a/sabnzbd/__init__.py#L492